modules/ipi-install-network-requirements.adoc

```diff
@@ -160,5 +160,5 @@ You can reconfigure the control plane nodes to act as NTP servers on disconnecte
 
 [id='network-requirements-out-of-band_{context}']
 == Port access for the out-of-band management IP address
-
-The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address must be granted access to port `6180`.
+`
+The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address must be granted access to port `6180` on the OpenShift control plane hosts, and port `80` on the bootstrap host.
```

Comment:
You could mention the bootstrap host first, since logically in an installation it is created before the control plane hosts. For example:

> To ensure that the out-of-band management can communicate with the bare metal node during installation, the out-of-band management IP address must be granted access to port `80` on the bootstrap host and port `6180` on the {product-title} control plane hosts.

It's just a preference of mine having worked on the installation assemblies that typically mention bootstrapping first.  So, please feel free to ignore this suggestion.